src/mod/update/v315/typedef315.go

```diff
@@ -59,7 +59,7 @@ type AuthProvider int
 const (
 	AuthProviderNone AuthProvider = iota
 	AuthProviderBasicAuth
-	AuthProviderAuthelia
+	AuthProviderForward
```

Comment:
I think this no need to be changed. Instead, a new v322/typedef321.go should be added to map `AuthProviderAuthelia` to `AuthProviderForward` (which can be added later after merge)